ext/openssl/openssl.stub.php

```diff
@@ -33,7 +33,10 @@ function openssl_x509_export(OpenSSLCertificate|string $certificate, &$output, b
 
 function openssl_x509_fingerprint(OpenSSLCertificate|string $certificate, string $digest_algo = "sha1", bool $binary = false): string|false {}
 
-/** @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key */
+/**
+ * @sensitive-param $private_key
```

Comment:
These are a bit unfortunate, since out of the `OpenSSLAsymmetricKey|OpenSSLCertificate|array|string` types, only the latter two are sensitive in fact (at least for now), and debugging becomes a bit more difficult in the rest of the cases. I don't think we can do better here, but wanted to point this out.

Reply:
Yes, agreed. This is OpenSSL's API shining through. I've opted to err on the side of "not redacting" to keep the impact low, but for this one the parameter definitely will see cryptographic secrets for often than not.

It's easy enough to add and remove this attribute in future versions. I would not consider either of those actions to be BC breaking, as stack trace consumers will need to be prepared for anything (or nothing) to have the attribute in userland code.

Reply:
Yes, agreed! I'm fine with keeping the attributes for these. Nevertheless, I asked a review from Jakub too, who is the primary maintainer of ext/openssl.

Reply:
In this case it should error / not find the key if you pass anything except the private key because `php_openssl_pkey_from_zval` is called with 0 as the second argument (not public key). So it seems correct to me.